priv/repo/migrations/20220121222855_create_wines.exs

```diff
@@ -0,0 +1,14 @@
+defmodule WineOInventoryApi.Repo.Migrations.CreateWines do
+  use Ecto.Migration
```

Comment:
I'd usually expect a migration to be its own PR

Reply:
Why do you think that is?

Reply:
I'm taking an educated guess here. I'm thinking it has to do with ensuring the migrations work correctly, before including code that interacts with the database?  Also, when working on a team, including migrations and code that interacts with a database that doesn't work correctly can cause problems if merged into master?

Reply:
Ding Ding Ding 🔔